.github/actions/install-dependencies/action.sh

```diff
@@ -6,6 +6,7 @@ python -m pip install --upgrade pip
 if [[ "${INSTALL_REQUIREMENTS}" == "true"  ]]; then
   echo "Installing code requirements"
   pip install -r requirements.txt
+  python -m spacy download en_core_web_sm #download spacy language pack
```

Comment:
lets do this in the `docker/Dockerfile` instead as the resources are needed beyond just the test / lint suite 👍